### DISCUSSION TITLE
Posted about it on communities, this project is far from discoverable

### DISCUSSION THREAD

**Discussion by Just-Moh-it** [imneonizer/autolycus#63] [Show and tell]
Hey, I wanted to have a self-hosted seedr-like website, but wasn't able to find any, but by chance got to this repo. More people need to know about it, through posts in communities. For the time being, I have posted about it on the self-hosted Reddit page [here](https://www.reddit.com/r/selfhosted/comments/svambx/autolycus_a_seedrcc_inspired_web_application_to/), and you can expect this repo gaining a lot of popularity in the short term @imneonizer maybe add it to [this list](https://github.com/awesome-selfhosted/awesome-selfhosted). Thanks

**Comment by imneonizer** [comment]
Thanks for the efforts!
